requirements.txt

```diff
@@ -2,6 +2,7 @@ aws-requests-auth==0.4.2
 boto==2.48.0
 boto3==1.7.48
 botocore==1.10.48
+commonmark==0.7.5
```

Comment:
Thank you for figuring this out, @Ben-Wu. The only thing I would have done differently is to add a comment above this line pointing at the GH issue that will make this workaround obsolete.